include/votca/csg/nblist_3body.h

```diff
@@ -0,0 +1,175 @@
+/* 
+ * Copyright 2009-2016 The VOTCA Development Team (http://www.votca.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef _NBLIST_3BODY_H
+#define	_NBLIST_3BODY_H
+
+#include "beadlist.h"
+#include "beadtriple.h"
+#include "triplelist.h"
+#include "exclusionlist.h"
+
+namespace votca { namespace csg {
+using namespace votca::tools;
+
```

Comment:
If possible remove using namespace votca::tools and use full scope in header file.